src/input/event/evdev.rs

```diff
@@ -617,6 +617,7 @@ fn event_codes_from_capability(capability: Capability) -> Vec<u16> {
                 GamepadButton::RightTop => vec![],
                 GamepadButton::RightTrigger => vec![KeyCode::BTN_TR2.0],
                 GamepadButton::Screenshot => vec![KeyCode::KEY_RECORD.0],
+                GamepadButton::Mute => vec![KeyCode::KEY_MUTE.0],
```

Comment:
Was this verified as the target event in evtest?

Reply:
In evtest, no event is emitted at all. Open to suggestions if we want this to translate to evdev at all.

Reply:
It think it's intended to be used with a headset attachment for the controller. Maybe we could only send this event out of the driver if no attachment is detected in the Input state.